py-polars/src/series/construction.rs

```diff
@@ -232,18 +232,24 @@ impl PySeries {
         _strict: bool,
     ) -> PyResult<Self> {
         let val = vec_extract_wrapped(val);
```

Comment:
This `val` should also in the second branch. We don't have to execute `val_extract_wrapped` if it is empty.

Reply:
Can we call `is_empty()` directly on a wrapped val or do I need to do `val.0.is_empty()`?

Reply:
Compiler answered my question.

Reply:
Ok done.